parts/k8s/kubernetescustomscript.sh

```diff
@@ -187,3 +188,4 @@ else
       aptmarkWALinuxAgent unhold &
   fi
 fi
+set_log_perms
```

Comment:
we move the /var/log directory in the Packer clean up step, we should do this after that is done. https://github.com/Azure/aks-engine/blob/master/packer/cleanup-vhd.sh#L11